tutorials/understanding-group/index.md

```diff
@@ -0,0 +1,327 @@
+In order to follow the tutorial, you must use the binary of a chain with the group module, using a v0.46+ version of the SDK.
+For demonstration purposes, we will use `simd`, the simulation app of the Cosmos SDK.
+
+For installing `simd`, first clone the Cosmos SDK GitHub repository:
+
+```sh
+git clone https://github.com/cosmos/cosmos-sdk --depth=1 
+```
+
+Go to the cloned directory and checkout the v0.46.0 release:
+
+```sh
+cd cosmos-sdk && git checkout v0.46.0
+```
+
+Install `simd`
+
+```sh
+make install
+```
+
+Make sure the installation was successful:
+
+```sh
+simd version
+```
+
+The version number should be greater than or equal to `0.46.0`.
+
+## Configuration
+
+::: tip
+If you have used `simd` before, you might already have a `.simapp` directory in your home directory. You can skip to the next section or remove the chain directory (`rm -rf ~/.simapp`).
+:::
+
+In order to configure `simd`, you need to set the chain ID and the keyring backend.
+
+```sh
+simd config chain-id demo
+simd config keyring-backend test
+```
+
+Secondly, you need to add keys for group users; let's call them alice and bob.
+
+```sh
+simd keys add alice
+simd keys add bob
+```
+
+With `simd keys list` you can verify that our two users have been added.
+
+::: tip
+To avoid having to copy and paste the user addresses, now is a good time to export the user keys to variables that you can access and use for this tutorial.
+:::
+
+```sh
+export ALICE_KEY=$(simd keys show alice -a)
+export BOB_KEY=$(simd keys show bob -a)
+```
+
+Now we are ready to fund alice and bob accounts and use alice account as validator:
+
+```sh
+simd init test --chain-id demo
+simd add-genesis-account alice 5000000000stake --keyring-backend test
+simd add-genesis-account bob 5000000000stake --keyring-backend test
+simd gentx alice 1000000stake --chain-id demo
+simd collect-gentxs
+```
+
+Lastly, we start the chain:
+
+```sh
+simd start
+```
+
+`simapp` is now configured and running! We can now play with the group module!
+
+## Create a group
+
+For creating a group, we must decide who is the admin and who are the members.
+All members have a voting weight that is used to calculate their voting power in the group.
+
+Let's create a `members.json` file that contains group members of a football association.
+Replace `aliceaddr` and `bobaddr` by the address of Alice (`$ALICE_KEY`) and Bob (`$BOB_KEY`).
+
+
+```json
+{
+    "members": [
+        {
+            "address": "aliceaddr",
+            "weight": "1",
+            "metadata": "president"
+        },
+        {
+            "address": "bobaddr",
+            "weight": "1",
+            "metadata": "treasurer"
+        }
+    ]
+}
+```
+
+
+```sh
+simd tx group create-group $ALICE_KEY "best football association" members.json
+```
+
+
+Let's see the group that we just created:
+
+```sh
+simd query group groups-by-admin $ALICE_KEY
+simd query group group-members 1 # use the group id given by the previous command
```

Comment:
I am thinking of:

1. Either split those 2 lines apart with some explanation text between.
2. And/Or add `--sync-mode block` to the `tx` command, and look at the id that was returned. And use this id by saying "confirm it is the same" and "reuse it". Because I always prefer to teach how to keep track of what you did instead of throwing an action to the wind and then using other commands to find out where it landed. Keep the query by admin because that's useful in its own right.